CONTRIBUTING.md

```diff
@@ -54,6 +54,28 @@ In addition to the style guide above, client-side Angular components should conf
 
 Projects can be manually linted with `npm run lint`. If you are using Atom and have run the [environment provisioning scripts](https://github.com/treehouseaustin/environment-setup) your IDE comes configured with AtomLinter which will use all project-specific rules. If you are using a different IDE you should configure it with appropriate linting and not rely on the CI process to catch code style issues.
 
+#### Sass-lint
+
+Projects with a css style component will additionally adhere to code style enforced by `sass-lint`. See the [.sass-lint.yml file](./.sass-lint.yml) and reference [sass-lint docs/rules](https://github.com/sasstools/sass-lint/tree/develop/docs/rules) for individual rule explanations.
+
+To include, update your `package.json` to include the following structure:
+```
+{
+  "devDependencies": {
+    "@treehouse/guidelines": "*"
+    "sass-lint": "*"
+  },
+  "sasslintConfig": "./node_modules/@treehouse/guidelines/.sass-lint.yml",
+  "scripts": {
```

Comment:
Let's keep the `scripts` configuration up to the project itself. It may be useful to `:` namespace especially on projects with both JS and frontend. (ex. `npm run lint`, `npm run lint:js` and `npm run lint:sass`)